builder/ebssurrogate/builder_acc_test.go

```diff
@@ -0,0 +1,135 @@
+package ebssurrogate
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	amazon_acc "github.com/hashicorp/packer-plugin-amazon/builder/ebs/acceptance"
+	"github.com/hashicorp/packer-plugin-sdk/acctest"
+)
+
+func TestAccBuilder_EbssurrogateBasic(t *testing.T) {
+	ami := amazon_acc.AMIHelper{
+		Region: "us-east-1",
+		Name:   fmt.Sprintf("packer-plugin-amazon-ebs-basic-acc-test %d", time.Now().Unix()),
+	}
+	testCase := &acctest.PluginTestCase{
+		Name:     "amazon-ebssurrogate_basic_test",
+		Template: fmt.Sprintf(testBuilderAccBasic, ami.Name),
+		Teardown: func() error {
+			return ami.CleanUpAmi()
+		},
+		Check: func(buildCommand *exec.Cmd, logfile string) error {
+			if buildCommand.ProcessState != nil {
+				if buildCommand.ProcessState.ExitCode() != 0 {
+					return fmt.Errorf("Bad exit code. Logfile: %s", logfile)
+				}
+			}
+			return nil
+		},
+	}
+	acctest.TestPlugin(t, testCase)
+}
+
+func TestAccBuilder_EbssurrogateBasic_forceIMDSv2(t *testing.T) {
+	aminame := fmt.Sprintf("packer-plugin-amazon-ebs-basic-acc-test %d", time.Now().Unix())
```

Comment:
I suggest giving making this unique as well to avoid any potential naming conflicts
```suggestion
	aminame := "ebsurrogate-acc-test-imdsv2"
```